apps/astarte_appengine_api/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hexpm/elixir:1.14.5-erlang-25.3.2-debian-bullseye-20230522-slim as builder
+FROM hexpm/elixir:1.15.5-erlang-24.3.4.9-debian-bookworm-20230612-slim as builder
```

Comment:
This is changing both Elixir and OTP version, other than changing the distro. I would expect this line to be something like
```
FROM hexpm/elixir:1.14.5-erlang-25.3.2-debian-bookworm-YYYYMMDD-slim as builder
```